pkgs/top-level/python-packages.nix

```diff
@@ -2843,20 +2843,39 @@ in {
       homepage = https://github.com/gitpython-developers/gitdb;
       license = licenses.bsd3;
     };
+  };
+
+  gitdb2 = buildPythonPackage rec {
```

Comment:
We are moving Python expressions out of `pkgs/top-level/python-packages.nix` into `pkgs/development/python-modules/<module>/default.nix`. 

Please move the expression there, and call it from `pkgs/top-level/python-packages.nix` using `callPackage ../development/python-modules/<package> { };`.